starport/services/chain/build.go

```diff
@@ -158,7 +158,8 @@ func (c *Chain) buildProto(ctx context.Context) error {
 	}
 
 	// If proto dir exists, compile the proto files.
-	if _, err := os.Stat(conf.Build.Proto.Path); os.IsNotExist(err) {
+	protoPath := filepath.Join(c.app.Path, conf.Build.Proto.Path)
+	if _, err := os.Stat(protoPath); os.IsNotExist(err) {
```

Comment:
Same as above, we must return an error if we encounter an error which is not `IsNotExist`

Reply:
removed by 0dbe9de.